request_a_govuk_domain/request/admin/views.py

```diff
@@ -53,6 +53,8 @@ def post(self, request):
                 self.message_user(
                     request, f"{approval_or_rejection} email sent", messages.SUCCESS
                 )
+                obj = Application.objects.get(pk=request.GET.get("obj_id"))
+                LOGGER.info(f"Reference {obj.reference}")
```

Comment:
Could we change this to 
"Application {obj.reference} status set to {approval_or_rejection}"